### PR TITLE
Fix appointment command error bugs

### LINKED ISSUE #272: [PE-D] Typos in error message for edit appointment command
Missing space between "INDEX" and "n/NAME" under parameters 

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: HuaiZe/ped#8

### PR DESCRIPTION
**Pull request by dvdweien**
fixes #272 #273

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java [hunk 1]
```diff
@@ -27,12 +27,12 @@
     public static final String COMMAND_WORD = "edit appointment";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits an appointment in the app. \n"
-            + "Parameters: INDEX"
+            + "Parameters: INDEX "
             + PREFIX_NAME + "NAME "
             + PREFIX_REMARK + "REMARK "
             + PREFIX_DATE + "DATE "
             + PREFIX_TIME + "TIME \n"
```
Comment (w2vgd) [inline src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java]: Do you need to add the square brackets here too?

**Review by w2vgd — COMMENTED** [review]
Should there be square brackets for the parameters?

File: src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java (hunk 1 above)
Comment (dvdweien) [reply src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java]: oh yeah! nice! thanks! added it!

**Review by w2vgd — COMMENTED** [review]
LGTM

**Review by w2vgd — APPROVED** [review]
LGTM

**Pull request merged**